cookbook/cookbook.md

```diff
@@ -3,20 +3,20 @@
 Creating an API provider:
 
 ```js
-import { ApiNetworkProvider } from "@multiversx/sdk-network-providers";
+import { ApiNetworkProvider } from "@multiversx/sdk-core";
 
 const apiNetworkProvider = new ApiNetworkProvider("https://devnet-api.multiversx.com", { clientName: "multiversx-your-client-name" });
 ```
 
 Creating a Proxy provider:
 
 ```js
-import { ProxyNetworkProvider } from "@multiversx/sdk-network-providers";
+import { ProxyNetworkProvider } from "@multiversx/sdk-core";
 
 const proxyNetworkProvider = new ProxyNetworkProvider("https://devnet-gateway.multiversx.com", { clientName: "multiversx-your-client-name" });
 ```
 
-Use the classes from `@multiversx/sdk-network-providers` **only as a starting point**.
+Use the classes from `@multiversx/sdk-core/out/networkProviderss` **only as a starting point**.
```

Comment:
Typo: `networkProviderss`. `out` should be removed, as well.

Reply:
done